.storybook/.babelrc

```
@@ -1,3 +1,6 @@
 {
-  "presets": [["@dsmjs", {"targets": {"browser": true}, "browser": true, "react": true}]]
+  "presets": [
+    ["@dsmjs", {"targets": {"browser": true}, "browser": true, "react": true}],
```

totally fine to be part of a follow-up PR, but would you mind removing the extra `browser: true` from the config? not sure how that got in there, but it should be safe to remove while we're updating things in here

```suggestion
    ["@dsmjs", {"targets": {"browser": true}, "react": true}],
```

---

Okay, not being too familiar with these configs I didn't really look for anything to clean up. 

Do you know of any documentation on what this `"browser: true` setting does and the difference between the objects `{ key: value}` syntax vs the setting without curly braces? 

Looking https://babeljs.io/docs/en/6.26.3/babelrc and other docs on their site isn't helping.

---

you're doing a great job highlighting where i've slacked on documentation, haha. 

short answer, we're using a custom babel-preset intended for projects under the dsmJS org. longer answer, i was going to send a link to the [org preset](https://github.com/dsmjs/babel-preset), which should be documenting basic usage, but in doing so i haven't updated the docs of that preset yet. not a huge deal, that preset extends a [base preset](https://github.com/form8ion/babel-preset) that i created to share the core details among a few different account-level presets. unfortunately, i haven't updated the docs there either 🤦‍♂ 

so, apparently the best i can offer for now are the other presets that extend the core ones:
* https://github.com/travi/babel-preset-travi
* https://github.com/gaincompliance/babel-preset-gain

hopefully, that helps to answer some of the questions you ran into, but i'm certainly open to feedback about how it could be improved. also, thanks for the nudge to get the others updated.

all that said, i don't mean to distract from the target of getting emotion, so don't hesitate to push back if i mention things like this that seem like to big of a distraction to tackle along the way.